spectral/ruleset.yaml

```diff
@@ -9,6 +9,21 @@ functions:
 rules:
   openapi-tags: off
   operation-tags: off
+  # openapi spec rules, that return errors
+  oas3-valid-schema-example: off
+  no-$ref-siblings: off
+  # openapi spec rules, that return warnings
+  operation-description: off
```

Comment:
This is trivial fix that we should enforce.

Reply:
yes.. But that is wrong way to fix it. Ruleset should stay as it is. If we do not want to fix anything we should add exceptions per service in their spectral.yaml as suggested to RHOSE etc.

Reply:
ok I got you now. I thought you wanted our ruleset changed to pass. thats what I got from the chat. no worries. I can sort this.